File: EmployeeManagement/WebApplication1/Enums/Team.cs
using System.ComponentModel;

namespace EmployeeManagement.Enums
{
    public enum Team
    {
        [Description("Mobile")]
        Mobile = 1,
        [Description("Front end")]
        FrontEnd = 2,
        [Description("Back end")]
        BackEnd = 3,
        [Description("Not required")]
        NotRequired = 4,
    }
}

File: employee-management/src/utils/team.ts
export const team = (value : number) => {
  switch (value) {
    case 1:
      return "Mobile";
    case 2:
      return "Front end";
    case 3:
      return "Back end";
    case 4:
      return "Not required";
    default:
      return "";
  }
}
